Kitodo-DataAccess/pom.xml

```diff
@@ -44,6 +44,11 @@
             <artifactId>log4j-slf4j-impl</artifactId>
             <version>2.9.1</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-collections4</artifactId>
+            <version>4.1</version>
+        </dependency>       
```

Comment:
Please remove the white spaces at the end of the line. This was only visible after run a `git diff master...<your_branch>` on console.